### PR TITLE
Documentation update for Remote Environment improvements

### LINKED ISSUE #847: Provide documentation about new RE creation process.
**Description**

The Remote Environment controller will take care of provisioning a new Remote Environments. There will be no default Remote Environment available and user needs to create one using  Console UI or the Kubectl.

**Reasons**

This is done as improvements in Remote Environment.

### PR DESCRIPTION
**Pull request by lszymik**
**Description**

Changes proposed in this pull request:

- Documentation update for Remote Environment improvements

**Related issue(s)**
Resolves #847

### CODE REVIEW

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 1]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
 type: Details
 ---
 
-By default, Kyma comes with two Remote Environments preinstalled. Those Remote Environments are installed in the `kyma-integration` Namespace.
+The Remote Environment controller is responsible for managing all Remote Environment.
```
Comment (janmedrek) [inline docs/application-connector/docs/010-details-ac-deployment.md]: As above, it is rather responsible for managing RE infrastructure and not REs themselves

File: docs/application-connector/docs/040-cr-remote-evironment.md [hunk 2]
```diff
@@ -55,17 +51,18 @@ This table lists all the possible parameters of a given resource together with t
...
 | **spec.source** |    **NO**   | Identifies the Remote Environment in the cluster. |
-| **spec.source.environment** |    **YES**   | Specifies the environment of the connected Remote Environment. |
-| **spec.source.type** |    **YES**   | Specifies the type of the connected Remote Environment. |
-| **spec.source.namespace** |    **YES**   | Specifies the namespace of the connected Remote Environment. |
 | **spec.description** |    **NO**   | Describes the connected Remote Environment.  |
 | **spec.accessLabel** |    **NO**   | Labels the environment when the [EnvironmentMapping](041-cr-environment-mapping.md) is created. |
+| **spec.labels** |    **NO**   | Labels indentified Remote Environment taxonomy.  |
 | **spec.services** |    **NO**   | Contains all services that the Remote Environment provides. |
 | **spec.services.id** |    **YES**   | Identifies the service that the Remote Environment provides. |
+| **spec.services.identifier** |    **NO**   | Additional identifier of the service class. |
```
Comment (janmedrek) [inline docs/application-connector/docs/040-cr-remote-evironment.md]: Regarding structure: services also have `name` field (used by service-catalog I think), it is filled by metadata service during registration.

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 3]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
 
-To add a new Remote Environment to the cluster, run this command:
+ - Click Create and the new Remote Environment will be created and you can check status in Remote Environment view.
+ 
 
-```
-helm install --name {remote-environment-name} --set deployment.args.sourceType=commerce --set global.isLocalEnv=false --set global.domainName={domain-name} --namespace kyma-integration ./resources/remote-environments
+### Using Kubectl
+
+New Remote Environment can be created using the Kubectl apply command with the following yaml file:
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: ``You can create a new Remote Environment using the kubectl `apply` command for the `re-production-1.yaml` file:``

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 4]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+
+re-production-1.yaml:
+
+``` yaml
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: RemoteEnvironment
+metadata:
+  name: production-1
+spec:
+  description: This Remote Environment for connecting production system 1.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: ``This is a Remote Environment for connecting production system 1.``

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 5]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+  labels:
+    region: us
+    kind: production
 ```
 
-The **global.domainName** is mandatory. Example values can look like:
+Then you can execute the following command:
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `Run the following command:`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 6]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+
+``` bash
+kubectl apply -f ./re-production-1.yaml
 ```
-wormhole.cluster.kyma.cx
-nightly.cluster.kyma.cx
+
+## Delete a Remote Environment
+
+The Remote Environment can be removed from Kyma using either Console UI or Kubectl.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: ``You can remove a Remote Environment from Kyma using either the Console UI or kubectl.``

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 7]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+
+### Using the UI:
+
+- Go to the Kyma console UI.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `Console`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 8]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+- Select Administration.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `**Administration**`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 9]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+- Select the Remote Environments from the Integration section.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: ``Select the **Remote Environments** from the **Integration** section.``

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 10]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+- Choose the Remote Environment to which you want to delete.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `Choose the Remote Environment` ~to which~ `you want to delete.`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 11]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+- Click Delete.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: ``- Click **Delete**.``

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 12]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+
+![Delete RE](./assets/delete-re.png)
+ 
+
+### Using Kubectl
+
+The Remote Environment can be deleted using the following command:
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `Delete the Remote Environment using the following command:`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 13]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+
+Then you can execute the following command:
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: ~Then you can execute the following command:~

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 14]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+- Go to the Kyma console UI.
+- Select Administration.
+- Select the Remote Environments from the Integration section.
+- Choose the Remote Environment to which you want to delete.
+- Click Delete.
+
+![Delete RE](./assets/delete-re.png)
+ 
+
+### Using Kubectl
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `kubectl`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 15]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+The Remote Environment can be deleted using the following command:
+
+Then you can execute the following command:
+
+```bash
+kubectl delete re name-of-remote-environment
 ```
 
-You can override the following parameters:
+## Update Remote Environment
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `a Remote Environment`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 16]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
 
-- **sourceEnvironment** is the Event source Environment name.
-- **sourceType** is the Event source type.
-- **sourceNamespace** is the organization that publishes the Event.
+The Remote Environment can be updated using either Console UI or Kubectl.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: ``You can update a Remote Environment using either the Console UI or kubectl.``

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 17]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
 
-## Work with Helm
+>**NOTE:** The name of Remote Environment cannot be changed.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `You cannot change the name of a Remote Environment.`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 18]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
-- `helm list` lists existing Helm releases
-- `helm test [release-name]` tests a release
-- `helm get [release-name]` shows the contents of `.yaml` files that make up the release
-- `helm status [release-name]` shows the status of a named release
-- `helm delete [release-name]` deletes a release from Kubernetes
+### Using the UI:
 
-The full list of the Helm commands is available in the [Helm documentation](https://docs.helm.sh/helm/).
-You can also use the `helm --help` command.
+- Go to the Kyma console UI.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `Console`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 19]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+- Select Administration.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: ``- Select **Administration**.``

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 20]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+- Select the Remote Environments from the Integration section.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: ``Select the **Remote Environments** from the **Integration** section.``

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 21]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+- Choose the Remote Environment to which you want to update.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `Choose the Remote Environment` ~to which~ `you want to update.`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 22]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+- Change the description and labels
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: Dot at the end

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 23]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+- Click Save.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `- Click **Save**.`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 24]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+
+### Using Kubectl
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `kubectl`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 25]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+
+Update the file: `re-production-1.yaml`
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: ``Update the `re-production-1.yaml` file:``

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 26]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+
+``` yaml
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: RemoteEnvironment
+metadata:
+  name: production-1
+spec:
+  description: This is new description.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `This is a new description.`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 27]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+  labels:
+    region: new-region
+    kind: production
+```
+
+Then you can execute the following command:
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: `Run the following command:`

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 28]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
+
+``` bash
+kubectl apply -f ./re-production-1.yaml
+```
 
-## Use kubectl
+## How to check if your remote environment was successfully created.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: I'd move this info at the end of the "Install a Remote Environment" section. I wouldn't make a separate section our of it.

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 29]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
...
 
-To check if everything runs correctly, use the `kubectl get pods -n kyma-integration` or `kubectl get services -n kyma-integration` command.  
+The new Remote Environment will appear on the Remote Environment lists and will have status `Serving.`
```
Comment (klaudiagrz) [inline docs/application-connector/docs/010-details-ac-deployment.md]: ``The new Remote Environment appears on the **Remote Environments** list with the `Serving` status.``

File: docs/application-connector/docs/040-cr-remote-evironment.md [hunk 30]
```diff
@@ -55,17 +51,19 @@ This table lists all the possible parameters of a given resource together with t
...
```
Comment (klaudiagrz) [inline docs/application-connector/docs/040-cr-remote-evironment.md]: `Provides an additional identifier of the ServiceClass.`

File: docs/application-connector/docs/040-cr-remote-evironment.md [hunk 31]
```diff
@@ -55,17 +51,19 @@ This table lists all the possible parameters of a given resource together with t
 |:----------:|:-------------:|:------|
 | **metadata.name** |    **YES**   | Specifies the name of the CR. |
 | **spec.source** |    **NO**   | Identifies the Remote Environment in the cluster. |
-| **spec.source.environment** |    **YES**   | Specifies the environment of the connected Remote Environment. |
-| **spec.source.type** |    **YES**   | Specifies the type of the connected Remote Environment. |
-| **spec.source.namespace** |    **YES**   | Specifies the namespace of the connected Remote Environment. |
 | **spec.description** |    **NO**   | Describes the connected Remote Environment.  |
 | **spec.accessLabel** |    **NO**   | Labels the environment when the [EnvironmentMapping](041-cr-environment-mapping.md) is created. |
+| **spec.labels** |    **NO**   | Labels indentified Remote Environment taxonomy.  |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/040-cr-remote-evironment.md]: `Labels the identified Remote Environment's taxonomy.` - is that what you mean?

File: docs/application-connector/docs/040-cr-remote-evironment.md [hunk 32]
```diff
@@ -55,17 +51,19 @@ This table lists all the possible parameters of a given resource together with t
...
 | **spec.services** |    **NO**   | Contains all services that the Remote Environment provides. |
 | **spec.services.id** |    **YES**   | Identifies the service that the Remote Environment provides. |
+| **spec.services.identifier** |    **NO**   | Additional identifier of the service class. |
+| **spec.services.name** |    **NO**   | Unique name of the service used by the Service Catalog. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/040-cr-remote-evironment.md]: `Represents a unique name`...

File: docs/application-connector/docs/040-cr-remote-evironment.md [hunk 33]
```diff
@@ -55,17 +51,19 @@ This table lists all the possible parameters of a given resource together with t
...
 | **spec.services.displayName** |    **YES**   | Specifies a human-readable name of the Remote Environment service. |
+| **spec.services.description** |    **NO**   | Provides a short human-readable description of the Remote Environment service. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/040-cr-remote-evironment.md]: ...`a short, human-readable`...

File: docs/application-connector/docs/010-details-ac-deployment.md [hunk 34]
```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
 type: Details
```
[the remark this answers is not in the document]
Comment (lszymik) [reply docs/application-connector/docs/010-details-ac-deployment.md]: It is part of the bigger documentation refactor.

**Pull request merged**